application/comit_node/src/swap_protocols/rfc003/ethereum/contract_templates/ether_contract.asm

```diff
@@ -2,6 +2,15 @@
     // Placeholder for deployment timestamp
     0x50000005
 
+    // Load expect secret size
+    0x6006
```

Comment:
❓why not using the same placeholder as in ERC20 contract (`0x7007`)?

Reply:
+1 on hardcoding it.  This would resolve the issue I commented below as well (the one with `calldatacopy`)

Reply:
@bonomat because 6 is already used in Ether contract

@LLFourn how is it unsound if the hex is committed?

Reply:
I don't see how you would do it without placeholders @LLFourn and I would still expect the build to be done automatically from EVM to hex.
Let's see how #606 is implemented.

Reply:
Please note I removed the placeholders for the secret size.